install/kubernetes/helm/istio/values-istio-gateways.yaml

```diff
@@ -0,0 +1,325 @@
+# Common settings.
```

Comment:
You don't need to cut&paste the entire file - just override.

Reply:
I can clean up later, good point.